packages/core/src/components/SupportButton/SupportButton.tsx

```diff
@@ -111,53 +97,20 @@ PropsWithChildren<Props>) => {
               {child}
             </ListItem>
           ))}
-          {/* {tags && tags.length > 0 && (
-            <ListItem alignItems="flex-start">
-              <StackOverflow>
-                {tags.map((tag, i) => (
-                  <StackOverflowTag key={i} tag={tag} />
-                ))}
-              </StackOverflow>
-            </ListItem>
-          )} */}
-          {slackChannels && (
-            <ListItem>
-              <ListItemIcon>
-                <GroupIcon />
-              </ListItemIcon>
-              <ListItemText
-                disableTypography
-                primary={<Typography>Support</Typography>}
-                secondary={
-                  <div>
-                    {slackChannels.map((channel, i) => (
-                      <Link key={i}>{channel}</Link>
-                    ))}
-                  </div>
-                }
-              />
-            </ListItem>
-          )}
-          {contactEmails.length > 0 && (
```

Comment:
had never seen seen this before 😁

Reply:
Didn't seem to be used anywhere. I've made this more generalized now with the `SupportItem` structure in the config.